engine/src/mblandroidmisc.cpp

```diff
@@ -802,6 +802,26 @@ bool MCSystemBuildInfo(MCStringRef p_key, MCStringRef& r_value)
 
 ////////////////////////////////////////////////////////////////////////////////
 
+bool MCSystemRequestPermission(MCStringRef p_permission, bool& r_granted)
+{
+    r_granted = MCAndroidCheckRuntimePermission(p_permission);
```

Comment:
@livecodepanos you need to add do-nothing stubs for these to `mbliphonemisc.mm` I think